### DISCUSSION TITLE
BM25 vectors

### DISCUSSION THREAD

**Discussion by chenyn66** [castorini/pyserini#1294] [Q&A] [answered]
Hi,
I wonder if the BM25 vectors computed by `dump_documents_BM25` are already multiplied by the IDF of each term? 
Thanks

**Comment by MXueguang** [comment, marked as answer]
yes it should be

**Reply by lintool** [reply]
Yes, they are the BM25-weighted vectors, which has all components of the score, including the IDFs.
